kim_convergence/batch.py

```
@@ -109,31 +110,36 @@ def batch(time_series_data: Union[np.ndarray, List[float]],
+                f"scale is not a `str`.\nScale = {scale} is not a valid "
+                "method to scale and standardize a dataset."
             )
 
         if scale not in scale_methods:
             raise CRError(
                 f'method "{scale}" not found. Valid methods to scale and '
-                'standardize a dataset are:\n\t- '
-                + "\n\t- ".join(scale_methods)
+                "standardize a dataset are:\n\t- "
+                f'{"\n\t- ".join(scale_methods)}'
             )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)